models/BaseLucee.cfc

```diff
@@ -911,6 +911,8 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 		switch( driverName ) {
 			case 'MSSQL' :
 				return 'com.microsoft.jdbc.sqlserver.SQLServerDriver';
+			case 'MSSQL2' :
```

Comment:
With the translation fix, this case would actually be `JTDS` since acts on the "Lucee version" of the name.